src/sagemaker_containers/environment.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import logging
+import multiprocessing
+import os
+import shlex
+import subprocess
+import sys
+
+BASE_PATH = '/opt/ml'
+
+logger = logging.getLogger(__name__)
+
+_CODE_BASE_NAME = 'code'
+_MODEL_BASE_NAME = 'model'
+
+
+# TODO (mvsusp) -  create a shortcut decorator to allow get/set outside the class, e.g. env.model_dir = 'x'
+base_dir = BASE_PATH
```

Comment:
why are we exposing as model var at all? it's already a class attribute.

Reply:
It is temporary, just to reduce the size of the PR. But I added a TODO in the top to fix it.